ckan/tests/controllers/test_organization.py

```diff
@@ -209,6 +210,25 @@ def test_anon_user_trying_to_delete_fails(self):
                                            id=self.organization['id'])
         assert_equal(organization['state'], 'active')
 
+    def test_delete_organization_with_datasets(self):
+        ''' Test deletion of organization that has datasets'''
+        config['ckan.auth.create_unowned_dataset'] = False
```

Comment:
Use the `@change_config` decorator so the value is reset when the test is finished